docs/articles/documentation/test-api/actions/screenshot.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: docs
+title: Screenshot Action
+permalink: /documentation/test-api/actions/screenshot.html
+---
+# Screenshot Action
+
+Takes a screenshot of the tested page.
+
+```text
+takeScreenshot( [path] )
+```
+
+Parameter           | Type   | Description                                                                                           | Default
+------------------- | ------ | ----------------------------------------------------------------------------------------------------- | ----------
+`path` *(optional)* | String | Relative path to the folder where screenshots should be saved resolved from the *screenshot directory* specified via an [API](../../using-testcafe/programming-interface/runner.md#screenshots) or [CLI](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) option. | Screenshot directory specified via an [API](../../using-testcafe/programming-interface/runner.md#screenshots) or [command-line interface](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) option.
```

Comment:
> Relative path to the folder where screenshots should be saved resolved from the *screenshot directory* specified via an [API]

1) I would divide this into two sentences.
2) An article is required before _relative path_